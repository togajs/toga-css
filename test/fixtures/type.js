/** @param {Type} */
/** @param {String|Object} */
/** @param {Array.<Object.<String,Number>>} */
/** @param {Function(String, ...[Number]): Number} callback */
