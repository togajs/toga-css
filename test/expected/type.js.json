{
    "type": "Tunic",
    "blocks": [
        {
            "type": "Code",
            "contents": ""
        },
        {
            "type": "Comment",
            "description": "",
            "tags": [
                {
                    "tag": "param",
                    "type": "{Type}"
                }
            ]
        },
        {
            "type": "Code",
            "contents": ""
        },
        {
            "type": "Comment",
            "description": "",
            "tags": [
                {
                    "tag": "param",
                    "type": "{String|Object}"
                }
            ]
        },
        {
            "type": "Code",
            "contents": ""
        },
        {
            "type": "Comment",
            "description": "",
            "tags": [
                {
                    "tag": "param",
                    "type": "{Array.<Object.<String,Number>>}"
                }
            ]
        },
        {
            "type": "Code",
            "contents": ""
        },
        {
            "type": "Comment",
            "description": "",
            "tags": [
                {
                    "tag": "param",
                    "type": "{Function(String, ...[Number]): Number}",
                    "name": "callback"
                }
            ]
        },
        {
            "type": "Code",
            "contents": ""
        }
    ]
}
